python_modules/dagster/dagster/_core/definitions/input.py

```diff
@@ -372,13 +372,7 @@ class InputMapping(
         [("definition", InputDefinition), ("maps_to", Union[InputPointer, FanInInputPointer])],
     )
 ):
-    """Defines an input mapping for a composite solid.
-
-    Args:
-        definition (InputDefinition): Defines the input to the composite solid.
-        solid_name (str): The name of the child solid onto which to map the input.
-        input_name (str): The name of the input to the child solid onto which to map the input.
-    """
+    """Defines an input mapping for a graph."""
```

Comment:
is args removal intentional here?

Reply:
Yeah - two reasons:
- The documented args aren't actually accurate
- We don't want people to be constructing these, because the only acceptable values are a legacy type (InputDefinition) and a non-public type